README.md

```diff
@@ -204,66 +206,112 @@ const toast = document.createElement('std-toast');
 console.log(toast.open); // false
 ```
 
+There will additionally be an `action` property,
+which returns or allows setting an element that provides the toast's action.
+In detail:
+
+- The getter returns the first `button` or `a` element descendant with the `slot="action"` attribute set.
+  If no such element exists,
+  it returns null.
+- The setter accepts a `button` or `a` element,
+  appends it to the toast element,
+  and sets the `slot="action"` attribute on it.
+  If any existing such elements exist,
+  the first one (i.e. the one that would be returned by the getter) is replaced.
+  If the provided value is not a `HTMLButtonElement` or `HTMLAnchorElement`,
+  a `TypeError` is thrown.
+
+To see examples of the `action` getter and setter in use,
+read on to the next section.
+
 #### Contents
 
-The `<std-toast>` element can be used as a generic container,
-but will work best (in terms of default styling and events) if the developer conforms to the following content model:
+At a technical level, the `<std-toast>` element can be used as a generic container.
+However, the authoring conformance requirements,
+as well as the API,
+steer the developer in to the following content model:
 
-- The first child node,
-  typically either a text node or a container like `<p>`,
-  provides a message.
-- If more than one child node is present,
-  then the last child element,
-  which can be either an `<a>` or a `<button>`,
-  provides a call to action.
+- All children except an action need to be [phrasing content](https://html.spec.whatwg.org/#phrasing-content-2)
+  that is not [interactive content](https://html.spec.whatwg.org/#interactive-content-2).
+  These children provide the toast's message.
+- There may be zero or one actions,
+  which are either `<a>` or `<button>` elements,
+  decorated with the `slot="action"` attribute.
 
-TODO([#17](https://github.com/jackbsteinberg/std-toast/issues/17)): what about title or icon? They should potentially also be accomodated, in a similar fashion.
+TODO([#17](https://github.com/jackbsteinberg/std-toast/issues/17)): what about title or icon?
+They could potentially also be accomodated, in a similar fashion.
 
-Thus, the following would all work well out of the box:
+Thus,
+the following would all work well out of the box,
+and be comformant to the content model:
 
 ```html
 <std-toast>Hello world!</std-toast>
 
-<std-toast><p>Hello world!</p></std-toast>
+<std-toast><p>Hello <strong>world</strong>!</p></std-toast>
 
-<std-toast>
-  <p>Hello world!</p>
-  <button>Click me!</button>
+<std-toast id="toast3">
```

Comment:
I'm also a bit confused, are these meant to be the result of the JS below, or just a parallel way of doing it? If the first thing, maybe just add comments above each of the HTML representations matching them to the # of toast in the js, this one has an ID but the others don't. Otherwise it might make sense to move the snippet query selecting this element up next to it.

Reply:
It's meant to be parallel, per the "These can equivalently be created via JavaScript:". I'll move the snippet up next to this; let me know if you have other thoughts on how to make it clearer.